src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import EditIcon from '@mui/icons-material/Edit'
+import AddIcon from '@mui/icons-material/Add'
+
+import AppButton from '~/components/app-button/AppButton'
+import AppTextField from '~/components/app-text-field/AppTextField'
+import PageWrapper from '~/components/page-wrapper/PageWrapper'
+
+import { myResourcesPath } from '~/pages/create-or-edit-lesson/CreateOrEditLesson.constants'
+import { styles } from '~/containers/my-quizzes/edit-quiz-container/EditQuizContainer.styles'
+import {
+  ButtonTypeEnum,
+  ButtonVariantEnum,
+  ComponentEnum,
+  SizeEnum,
+  TextFieldVariantEnum
+} from '~/types'
+
+const EditQuizContainer = () => {
+  const { t } = useTranslation()
+  const [title, setTitle] = useState<string>('')
+  const [description, setDescription] = useState<string>('')
+
+  const handleTitleChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setTitle(e.currentTarget.value)
+  }
+
+  const handleDescriptionChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setDescription(e.currentTarget.value)
+  }
+
+  return (
+    <PageWrapper sx={styles.container}>
+      <Box component={ComponentEnum.Form} sx={styles.root}>
+        <AppTextField
+          InputLabelProps={styles.titleLabel}
+          InputProps={styles.titleInput}
+          fullWidth
+          inputProps={styles.input}
+          label={title ? '' : t('myResourcesPage.quizzes.defaultNewTitle')}
```

Comment:
you can pass title to styles.titleLabel(title) and then in styles hide label,
visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,

Reply:
> you can pass title to styles.titleLabel(title) and then in styles hide label, visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,

thank you for your advice, slightly changed your approach and fixed it